server/build.gradle

```
@@ -17,6 +17,13 @@
  * under the License.
  */
 
+plugins {
+  id "com.github.hierynomus.license-report" version"0.15.0"
+}
+
+downloadLicenses {
+    dependencyConfiguration = 'bundleJars'
```

should we explicitly set `includeProjectDependencies` to `false`
```suggestion
    dependencyConfiguration = 'bundleJars'
    includeProjectDependencies = false
```